workloads/kube-burner/common.sh

```diff
@@ -54,17 +54,25 @@ collect_pprof() {
 
 run_workload() {
   local CMD
-  curl -sS -L ${KUBE_BURNER_URL} | tar -xzC /tmp/ kube-burner
+  local KUBE_BURNER_DIR 
+  KUBE_BURNER_DIR=$(mktemp -d)
+  if [[ -n ${BUILD_FROM_REPO} ]]; then
+    git clone --depth=1 ${BUILD_FROM_REPO} ${KUBE_BURNER_DIR}
+    make -C ${KUBE_BURNER_DIR} build
+    mv kube-burner/bin/amd64/kube-burner /tmp/kube-burner
```

Comment:
should be 
mv ${KUBE_BURNER_DIR}/bin/amd64/kube-burner /tmp/kube-burner

Reply:
fixed